podvm/Makefile.inc

```diff
@@ -18,7 +18,7 @@ HOST_ARCH   := $(shell uname -m)
 ARCH        ?= $(HOST_ARCH)
 # Normalise x86_64 / amd64 for input ARCH
 DEB_ARCH    := $(subst x86_64,amd64,$(ARCH))
-AA_KBC      ?= offline_fs_kbc
+AA_KBC      ?= offline_fs_kbc,cc_kbc
```

Comment:
Does a comma separated list of KBCs even work? We use this parameter to set `aa_kbc_params = ${AA_KBC}::${KBC_URI}` which surely would break and even the [attestation-agent Makefile](https://github.com/confidential-containers/guest-components/blob/main/attestation-agent/Makefile) doesn't seem to have support for multiple KBCs as it is doing equals checks rather than contains in it's processing and they don't seem to have any CI tests for this?

Reply:
The build seems to work, but you are right, it breaks when setting the agent config. As it cc_kbc also seems to not be supported on all platforms, I think we should remove that commit from this PR for now.

Reply:
cc @mkulke

Reply:
cc @mkulke

Reply:
https://github.com/confidential-containers/cloud-api-adaptor/pull/1211/commits/dc666c0c62f0050551d950d22894846f790a1c5a Removed from default, still installing the dependencies, so build works when setting `AA_KBC=cc_kbc`